gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcClientCalls.java

```diff
@@ -85,6 +86,8 @@
       channel = ClientInterceptors.intercept(channel, interceptor);
     }
 
-    return channel.newCall(descriptor, callOptions);
+    try (Scope ignored = grpcContext.getTracer().inScope()) {
```

Comment:
Just in case, is `grpcContext.getTracer()` always `!= null`?

Reply:
Yep